cspatients/templates/cspatients/patient.html

```diff
@@ -73,17 +73,21 @@ <h4>Surgery Details</h4>
                   <li class="list-group-item py-2">
                       <div style="width: 50%;float: left;">
                           DECISION TIME <br/>
-                          <div>{{ patiententry.decision_time|time:"H:i" }}</div>
+                          <div>{{ patiententry.decision_time|date:"Y-m-d" }} {{ patiententry.decision_time|time:"H:i" }}</div>
```

Comment:
@erikh360 just want to check - is it possible to have it  as `{{ patiententry.decision_time|date:"Y-m-d H:i" }}`?

Reply:
hehe, possibly